### PR TITLE
CRAN 1.0.5

### PR DESCRIPTION
**Pull request by mpadge**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -44,8 +50,8 @@ Description: Give advice about good practices when building R packages.
     Advice includes functions and syntax to avoid, package structure, code
     complexity, code formatting, etc.
 License: MIT + file LICENSE
-URL: https://mangothecat.github.io/goodpractice/, https://github.com/mangothecat/goodpractice
-BugReports: https://github.com/mangothecat/goodpractice/issues
+URL: https://ropensci-review-tools.github.io/goodpractice/, https://github.com/ropensci-review-tools/goodpractice
```
Comment (maelle) [inline DESCRIPTION:53]: why not docs.ropensci.org?

Comment (maelle) [reply DESCRIPTION:53]: https://github.com/ropensci/roregistry/blob/gh-pages/info/not_transferred.json

File: README.Rmd [hunk 2]
```diff
@@ -16,10 +16,10 @@ knitr::opts_chunk$set(
 # goodpractice <img src="man/figures/logo.png" align="right" width="20%" height="20%" />
 
   <!-- badges: start -->
-  [![R-CMD-check](https://github.com/MangoTheCat/goodpractice/workflows/R-CMD-check/badge.svg)](https://github.com/MangoTheCat/goodpractice/actions)
+  [![R-CMD-check](https://github.com/ropensci-review-tools/goodpractice/workflows/R-CMD-check/badge.svg)](https://github.com/ropensci-review-tools/goodpractice/actions)
   [![CRAN status](https://www.r-pkg.org/badges/version/goodpractice)](https://CRAN.R-project.org/package=goodpractice)
   [![CRAN RStudio mirror downloads](https://cranlogs.r-pkg.org/badges/goodpractice)](https://www.r-pkg.org/pkg/goodpractice)
-  [![Codecov test coverage](https://codecov.io/gh/MangoTheCat/goodpractice/branch/main/graph/badge.svg)](https://app.codecov.io/gh/MangoTheCat/goodpractice?branch=main)
+  [![Codecov test coverage](https://codecov.io/gh/ropensci-review-tools/goodpractice/branch/main/graph/badge.svg)](https://app.codecov.io/gh/ropensci-review-tools/goodpractice?branch=main)
```
Comment (maelle) [inline README.Rmd:22]: occasion to add an R-universe badge?

File: README.Rmd [hunk 3]
```diff
@@ -40,7 +40,7 @@ install.packages("goodpractice")
 and the development version from GitHub
 
 ```{r eval = FALSE}
-remotes::install_github("mangothecat/goodpractice")
+remotes::install_github("ropensci-review-tools/goodpractice")
```
Comment (maelle) [inline README.Rmd:43]:
```suggestion
pak::pak("ropensci-review-tools/goodpractice")
```

File: tests/testthat/test-print.R [hunk 4]
```diff
@@ -1,9 +1,8 @@
 
 test_that("print with default and explicit positions.limit", {
 
-  # These tests currently fail on Windows and Linux CI. Skipping for now...
-  skip_on_os("windows")
-  skip_on_os("linux")
+  # These tests currently fail on Windows CI. Skipping for now...
```
Comment (maelle) [inline tests/testthat/test-print.R:4]: why commenting rather than removing?

**Review by maelle — CHANGES_REQUESTED** [review]

File: DESCRIPTION (hunk 1 above)
Comment (mpadge) [reply DESCRIPTION:53]: https://github.com/ropensci/roregistry/pull/42 - do you want to merge that?

File: tests/testthat/test-print.R (hunk 4 above)
Comment (mpadge) [reply tests/testthat/test-print.R]: Nice catch; done now. Thanks!

File: README.Rmd (hunk 2 above)
Comment (mpadge) [reply README.Rmd:22]: Should really do that to all packages, so via a separate issue. I'd also like to keep this PR as minimal as possible.

File: DESCRIPTION (hunk 1 above)
Comment (mpadge) [reply DESCRIPTION:53]: Thanks for merging. Docs still won't be built until triggered by a change in main branch, and that won't happen until this PR is merged. My suggestion would be to first merge this, then subsequently update docs URL. Okay with you?

Comment (mpadge) [reply DESCRIPTION]: Done.

**Pull request merged**